main/resource-access.bs

```diff
@@ -75,6 +75,352 @@ Issue: Write Linked Data Platform section.
+on the resource directly associated with it.
+
+To discover or read a Descriptive auxiliary resource, an
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Read](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the
+[ACL inheritance algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+An Descriptive auxiliary resource MUST be deleted by the Solid server when the
+resource it is directly associated with is also deleted and the Solid server is
+authoritative for both resources.
+
+#### Shape Validation #### {#ar-shape}
+
+Shape Validation auxiliary resources as defined by (link to shape validation)
+SHOULD be supported as an auxiliary type by Solid servers.
+
+The Shape validation auxiliary resource directly associated with a given
+resource
+is discovered by the client via the `rel=http://www.w3.org/ns/solid/terms#shape`
+`Link` relation in a `Link` header. Conversely, the resource being described by
+a Shape validation auxiliary resource is discovered by the client via the
+`rel=describes` `Link` relation in a `Link` header.
+
+Note: Consider moving some of this information to the Shape Validation section
+
+A given Solid resource MUST NOT be directly associated with more than one
+Shape Validation auxiliary resource.
+
+Issue: Determine what the default permissions should be on shape validation
+auxiliary resources, or whether we should have them at all.
+[Related Issue](https://github.com/solid/specification/issues/174)
+
+To create or modify a Shape validation auxiliary resource, an
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Write](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the
+[ACL inheritance algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+To discover or read a Shape validation auxiliary resource, an
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Read](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the
+[ACL inheritance algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+A Shape validation auxiliary resource MUST be deleted by the Solid server when
+the resource it is directly associated with is also deleted and the Solid server
+is authoritative for both resources.
+
+Issue: Provide a shape to validate a shape validation auxiliary resource. May
+include the shape language, shape url, and any additional parameters to be used
+in shape validation by the server implementation.
+
+A Solid server SHOULD sanity check Shape validation auxiliary resources upon
+creation or update to restrict invalid changes.
+
+#### Server Managed #### {#ar-managed}
```

Comment:
What are differences between server-managed containment triples and the triples in server-managed resources (auxiliary)? It appears to be that we have two different mechanisms in the spec that are conceptually similar, if not the same. I suggest to at least partly address: https://github.com/solid/specification/issues/177 .

Reply:
Answered in #177

Reply:
Per the consensus in #177, server-managed as a broad catch-all has been removed from the proposal, and will be replaced with more specific relations in separate submissions.